act/act-mgr.go

```diff
@@ -0,0 +1,113 @@
+package act
+
+type Code int32
+type State uint8
+
+const cfgCodeMapSize = 16
+
+const KeyCode = Code(1 << 16)
+const MouseCode = Code(2 << 16)
+
+const StateIdle = State(0)
+const StateActive = State(1)
+const StateJustIdle = State(2)
+const StateJustActive = State(3)
+
+////////////////
```

Comment:
I disagree, from the go docs: 
> Comments that are not adjacent to a top-level declaration are omitted from godoc's output, with one notable exception.

So godoc allows the user to add top level comments that are ignored. If they are not ignored this is a bug or they need to adjust there documentation. Now I do understand you do not like this comment style, I remember that from the last pr and I was planning to remove it when it was ready to be merged.